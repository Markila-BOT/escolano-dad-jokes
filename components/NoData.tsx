import React from "react";

const NoData = () => {
  return (
    <div className="container mx-auto" data-testid="no-data-main">
      <h1 className="text-3xl font-bold text-center text-blue-400">
        No Jokes For You
      </h1>
    </div>
  );
};

export default NoData;
