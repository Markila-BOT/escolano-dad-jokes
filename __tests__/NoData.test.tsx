import { render, screen } from "@testing-library/react";
import NoData from "../components/NoData";

describe("NoData", () => {
  it("should renders the NoData component", () => {
    const { container } = render(<NoData />);
    expect(screen.getByTestId("no-data-main")).toBeTruthy();
    expect(container).toMatchSnapshot();
  });
});
