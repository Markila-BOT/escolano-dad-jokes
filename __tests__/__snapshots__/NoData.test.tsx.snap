// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoData should renders the NoData component 1`] = `
<div>
  <div
    class="container mx-auto"
    data-testid="no-data-main"
  >
    <h1
      class="text-3xl font-bold text-center text-blue-400"
    >
      No Jokes For You
    </h1>
  </div>
</div>
`;
